src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.Random.cs

```diff
@@ -3,13 +3,21 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
+
+        static Activity()
```

Comment:
I don't understand why we have two copies of the class constructor, but we want to get rid of both of them (see above)